OpenRA.Mods.Common/AI/BaseBuilder.cs

```diff
@@ -297,13 +328,18 @@ ActorInfo ChooseBuildingToBuild(ProductionQueue queue)
 				if (!buildableThings.Any(b => b.Name == name))
 					continue;
 
+				// Added new checks for if a building can be built and moved some older checks into it
+				// CTRLF - Check if the building has reached its limits
+				if (!BuildingIsBuildable(name))
```

Comment:
Ya it took a couple of tries to get it working when figuring out how that area of the code ran.  As for the AppVayor Error I don't get any errors on my end when compiling about the new class files so it might have to do with how I upload.  Either that or the error is going silent on my end one of the other.